db/migrate/20160605000844_add_notes_to_donors_to_doner.rb

```diff
@@ -0,0 +1,5 @@
+class AddNotesToDonorsToDoner < ActiveRecord::Migration
+  def change
+    add_column :donors, :notes_to_doner, :text
```

Comment:
Hmm, shouldn't this be a new column on the `donations` table classed `notes_to_donnor`?